### DISCUSSION TITLE
Today Win11 has been updated - ExplorerPatcher stopped working.

### DISCUSSION THREAD

**Discussion by VV-SH** [valinet/ExplorerPatcher#448] [General]
Subj

**Comment by BraINstinct0** [comment]
#440 
Read the issues first please.

**Reply by VV-SH** [reply]
BIG TNX

**Reply by BraINstinct0** [reply]
NP :) Glad I can help(or, at least let you know about the current situation)

**Reply by VV-SH** [reply]
With your help, I realized that I do not need to install Windows updates that are not automatically installed.
I deleted this update and everything works correctly.
